src/rnode-grpc.js

```diff
@@ -255,6 +255,23 @@ export const rnodeService = (options) => {
   )
```

Comment:
I'm ok to expose the client as part of current service but with underscore for now.
```js
const service = R.mapObjIndexed(
  createApiMethod({...options, protocol}, getType),
  methods,
)
return { ...service, _grpcClient: protocol.client }
```